traffic_portal/test/end_to_end/servers/servers-spec.js

```diff
@@ -67,6 +68,7 @@ describe('Traffic Portal Servers Test Suite', function() {
 		pageData.ipAddress.sendKeys(mockVals.ipAddress);
 		pageData.ipNetmask.sendKeys(mockVals.ipNetmask);
 		pageData.ipGateway.sendKeys(mockVals.ipGateway);
+		pageData.isServiceAddress.sendKeys(mockVals.isServiceAddress);
```

Comment:
ipIsService right?

Reply:
also, you're going to need an entry like this in the pagedata file

`	this.ipIsService=element(by.name('ipIsService'));
`

Reply:
also, i think you have to do

`pageData. ipIsService.click()`

Reply:
and you can get rid of the ipIsService: true,